src/main/java/org/apache/mesos/offer/DefaultOfferRequirementProvider.java

```diff
@@ -29,8 +29,9 @@ public DefaultOfferRequirementProvider(TaskConfigRouter taskConfigRouter) {
     @Override
     public OfferRequirement getNewOfferRequirement(String taskType, TaskSpecification taskSpecification)
             throws InvalidRequirementException {
+
         Protos.CommandInfo updatedCommand = taskConfigRouter.getConfig(taskType)
-                .updateEnvironment(taskSpecification.getCommand());
+                .updateEnvironment(taskSpecification.getCommand().get());
```

Comment:
Doesn't this need an `isPresent()` check before use?  If it's optional now we'll we want to skip adding it when it's not present correct?